### ISSUE TITLE
Power Project Tool Company

### ISSUE THREAD

**Issue by dtay2525** [dtay2525/PowerProjecttoolCo#1]
When first analyzing what needed to be completed for the Power Project company, I felt overwhelmed. However, taking my time and breaking down each individual step allowed me to create something I’m truly proud of completing. My first step was to create an html file and save it as PowerProjectKLE500.htm. I than began to construct the business website piece by piece starting with the <title> tag within the <head> also adding links using <link> tags to my created CSS style sheets named PowerProjectStyles.css & PowerprojectStyles1.css also linked a special font named 'Bruno Ace SC' from google fonts free font directory, also adding links to a potential logout screen.
	To begin the structure of the web page I began with the <body> tag and a <h1> header tag to create a visually appealing first glance. I decided to create a <nav> tag to place the home and Logout links on the left top. I decided I also wanted to allow users to have an option to sign in utilizing an email and password so I also added a register button to the <nav> tag to give a visual for new users to the website. When developing the login element, I used textboxes and checkboxes all within a <form> tag. 
	In the next section under the <h2> tag I created a header specific to the Klee500 Cordless tool series and structured the first paragraph in a <p> tag using and creating a class “big”. I then placed the features in a class of their own under “products” adding in each one of the images given for the new line. I ensured each image had an alternate text for the visually impaired as well. I then made a class specific to the features of the new line under class “features” using a unordered list with circle markings. 	
	In the next step I decided to create another <form> however this form would give users the ability to enter in first and last name, the product they may have questions or comments on and asking age range of submitters using the radio type and creating individual classes for each of the age ranges, this age inquiry will ultimately help with marketing possibilities in the future. The form will allow the user to be engaged and become part of the conversation.
 In the final section I needed to figure out how to incorporate a table with the proper dimensions and I figured what better way to showcase a table than by utilizing the contact information of the company by using the <table> tag and creating a class named “contact.”. I then constructed the table using the table row tag <tr> and the table heading tag <th> which placed a two row, three column table at the bottom center of the page.
In conclusion, although the designing of the webpage was difficult, I was eventually able to find a pattern that allowed me to create a website that has all the visual aspects needed to give the user a positive experience, from the creation of the register, email and password function to the ability to engage and locate beneficial comments and questions on the product information form. So, although it proved difficult at first to design, taking each step one by one helped me finalize a project I can be proud to turn in as Power Project Companies newest web page and product line.
